core/domain/classifier_services.py

```diff
@@ -329,15 +337,17 @@ def save_classifier_training_job(algorithm_id, exp_id, exp_version,
     Returns:
         job_id: str. ID of the classifier training job.
     """
-    classifier_training_job_model = (
-        classifier_models.ClassifierTrainingJobModel.get(job_id, False))
-    if classifier_training_job_model is None:
+    if status is None:
+        status = feconf.TRAINING_JOB_STATUS_NEW
```

Comment:
Done. I changed status to be called explicitly. We might have to check the job_id thingy.